py-polars/polars/dataframe/frame.py

```diff
@@ -4316,7 +4321,6 @@ def write_delta(
                 schema=schema,
                 mode=mode,
                 storage_options=storage_options,
-                large_dtypes=True,
```

Comment:
If version <= 0.22.3 we should still pass `large_dtypes=True`, I think.

Reply:
Actually, since v0.19 that parameter doesn't do anything when the Rust engine is used for writing, which is the default. I'll change the minimum in pyproject.toml to >=0.19